src/components/map/MapService.js

```diff
@@ -1530,20 +1530,39 @@ goog.require('ga_urlutils_service');
           return layer;
         },
 
+        flyToAnimation: function(ol3d, center, destination, defer) {
+          var scene = ol3d.getCesiumScene();
+          var camera = scene.camera;
+          var globe = scene.globe;
+          var carto = globe.ellipsoid.cartesianToCartographic(destination);
+          $http.get(gaGlobalOptions.apiUrl + '/rest/services/height', {
```

Comment:
@fredj on this [PR](https://github.com/geoadmin/mf-geoadmin3/pull/2861/files#diff-66ac86f99915855fde57c0a20d8f6d6bR1797) I use a cesium function to get the height of the Terrain without using the height service. Can you have a look a tell us which method we should use height service or Cesium function?